src/vfxt/src/createUiDefinition.json

```diff
@@ -202,7 +202,7 @@
               "name": "avereInstanceType",
               "type": "Microsoft.Compute.SizeSelector",
               "label": "Size",
-              "toolTip": "The SKU for the Avere vFXT VM.",
+              "toolTip": "The type of VM to use for the Avere vFXT cluster nodes.",
```

Comment:
I'd say, "The instance type used for..."

Reply:
updated